selfdrive/debug/uiview.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+import time
+import subprocess
+import cereal.messaging as messaging
+from common.basedir import BASEDIR
+
+pm = messaging.PubMaster(['controlsState', 'thermal'])
+subprocess.Popen(BASEDIR + "/selfdrive/camerad/camerad", cwd=BASEDIR + "/selfdrive/camerad")
+subprocess.Popen(BASEDIR + "/selfdrive/ui/ui", cwd=BASEDIR + "/selfdrive/ui")
```

Comment:
use manager helper functions

Reply:
and explicitly kill them on exit

Reply:
The only problem with that is importing anything from manager calls scons which takes upwards of 15 to 20 seconds before you're allowed to see the UI. I tried that with the driverview PR but that wouldn't be good since it's supposed to be callable in the UI

Reply:
I'd be happy to open a PR to move the scons code (from `# Run scons`) into the main() function so that it's possible to import manager helper functions and dicts/lists without a large delay though, just let me know.

Reply:
It doesn't really matter since this is a debug tool. Manager building at import time also slows down some of our tests, I'll refactor it soon.

Reply:
Ah okay, good to hear. I'll use those then. My reasoning was it can be annoying when all someone wants to do is see the UI, especially if you're actively adding UI elements and you just want to pull, compile the one file and see the updated UI quickly